azure-pipelines.yml

```diff
@@ -171,6 +171,22 @@ steps:
     platform: '$(buildPlatformNative)'
     configuration: '$(buildConfiguration)'
 
+# - task: CopyFiles@2
+#   inputs:
+#     sourceFolder: '$(sample3Folder)\src\windows\$(buildPlatformNative)\$(buildConfiguration)'
+#     contents: '**\$(sample3PublishedArtifactPath)\?(*.exe|*.dll|*.pdb)'
+#     targetFolder: '$(sample3PublishedArtifactPath)'
```

Comment:
No need ot keep this around if the  delete task is working for you

Reply:
Yes, removed, i left it there while i was testing